packages/common-all/data/dendron-yml.validator.json

```diff
@@ -647,8 +647,18 @@
             }
           },
           "additionalProperties": false
+        },
+        "aliasMode": {
+          "type": "string",
+          "enum": [
+            "none",
+            "alias"
```

Comment:
maybe this would be more intuitive if the enum value is called "title" instead of "alias"

Reply:
the reason we're using `alias` is because we currently use it in `insertNoteTitle`. would recommend changing that in a separate pass (eg. change both together) if we go down that route

Reply:
@kevinslin do you mean `insertNoteLink`? In that command, we have five options right now, once of which is `title`.

Reply:
I actually already changed `alias` to `title`. Should I change it back to `alias`?

Reply:
all good. lets keep it as is